tests/conftest.py

```diff
@@ -123,17 +125,17 @@ def user_cognito(username, jwt_partial_payload) -> dict:  # pylint: disable=rede
 
 
 @pytest.fixture(scope="session")
-def user_token(user_cognito) -> str:  # pylint: disable=redefined-outer-name
+def user_token(user_cognito: dict) -> str:  # pylint: disable=redefined-outer-name
```

Comment:
Why didn't you rename the `user_cognito` fixture? It would be good to remove these warnings as well 😉